js/components/closed-caption-multi-audio-menu/closedCaptionMultiAudioMenu.js

```diff
@@ -0,0 +1,128 @@
+var React = require('react');
+var ReactDOM = require('react-dom');
+var Utils = require('../utils');
+var classnames = require('classnames');
+var CONSTANTS = require('../../constants/constants');
+var Tab = require('./tab');
+var MultiAudioTab = require('./multiAudioTab');
+
+var ClosedCaptionMultiAudioMenu = React.createClass({
+  componentDidMount: function() {
+    var multiAudioCol = ReactDOM.findDOMNode(this.refs.multiAudioCol);
+    var closeCaptionsCol = ReactDOM.findDOMNode(this.refs.closeCaptionsCol);
+    if (multiAudioCol && closeCaptionsCol && typeof Utils.getMaxElementWidth === 'function') {
+      var maxWidth = Utils.getMaxElementWidth(multiAudioCol, closeCaptionsCol);
+      multiAudioCol.style.width = maxWidth + 'px';
+      closeCaptionsCol.style.width = maxWidth + 'px';
+    }
+  },
+  /**
+   *
+   * @param languages {Array} - list of available languages
+   * @param language {String} - a selected language
+   * @returns {Array<{id: String, label: String, enabled: Boolean}>} an array of languages info objects
+   * @private
+   */
+  getClosedCaptions: function(languages, language) {
+    var closeCaptions = [];
```

Comment:
See above comment on using closedCaptions instead of closeCaptions.

Reply:
\+